health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/GenericChart.js

```
@@ -0,0 +1,171 @@
+        case "shareImage":
+          return Digit.ShareFiles.IndividualChartImage(tenantId, chart, t(header), data.target);
+        default:
+          return null;
+      }
+    }, 1000);
+    setTimeout(() => {
+      {
+        if (setDownloadChartsId !== null) {
+          setDownloadChartsId(null);
+        }
+      }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)